editor/src/components/filebrowser/fileitem.tsx

```diff
@@ -278,6 +292,32 @@ class FileBrowserItemInner extends React.PureComponent<
 
   toggleCollapse = () => this.props.toggleCollapse(this.props.path)
 
+  renderGithubStatus = () => {
+    if (this.props.githubStatus != undefined) {
+      const c = this.props.githubStatus.at(0)?.toUpperCase()
```

Comment:
Super minor, but can you change the name of this const to `statusChar` please? I had to do a fair amount of looking about to figure out what this `c` actually represents to find out it is the first character of the git status